src/commands/authorizations/createAuthorizationAccessPolicy.ts

```diff
@@ -154,9 +155,10 @@ async function populateIdentityOptionsAsync(
 
     // 1. Self
     const token = await credential.getToken();
+    const decoded = jwt.decode(token.token, { complete: true }) as any;
```

Comment:
this bug is that azure account return token as a deviceTokenCredentials, in vscode session not support for this, so we need to decode the jwt token.